dockerfiles/base-images:centos7.9-1

```diff
@@ -0,0 +1,82 @@
+# syntax=docker/dockerfile:experimental
+FROM centos:centos7.9.2009
+
+# Install dependency packages
+# Install gcc & g++ ≥ 8 for Tesseract and Detectron2
+RUN yum -y update && \
+    yum -y upgrade && \
+    yum -y install poppler-utils xz-devel wget tar curl make which mailcap && \
+    yum -y install epel-release && \
+    yum -y install centos-release-scl && \
+    yum -y install devtoolset-9-gcc* && \
+    yum -y install pandoc && \
```

Comment:
If we are building pandoc further down, should it still be installed here?

Reply:
sorry forgot to remove that